apps/ai-manager-api/src/dependency-injection/services/gemini.ts

```diff
@@ -22,17 +29,29 @@ export const geminiService = {
       generationConfig: {
         temperature: 0.2,
         topP: 0.95,
-        maxOutputTokens: 1024,
         stopSequences: ['\n\n'],
         topK: 40,
+        responseMimeType: 'application/json',
+        responseSchema: {
+          type: SchemaType.OBJECT,
+          properties: {
+            text: {
```

Comment:
Here add or just return a json with the array of candidates, or optionally error, can be something